pkg/controller/restore_controller.go

```diff
@@ -406,8 +406,8 @@ func (c *restoreController) fetchBackupInfo(backupName string, pluginManager plu
 		}
 
 		log := c.logger.WithField("backupName", backupName)
-		log.Debug("Backup not found in backupLister, checking each backup location directly, starting with default...")
-		return c.fetchFromBackupStorage(backupName, pluginManager)
+		log.Debug("Backup not found in backupLister")
```

Comment:
L404-409 can be removed - any error when trying to get the backup from the lister should now be returned